collectors/UcsServerLicenseCollector.py

```diff
@@ -18,7 +18,8 @@ def collect(self):
         print("Getting updated handles for UcsServerLicenseCollector !")
         self.get_handles()
 
-        g = GaugeMetricFamily('ucs_port_license', 'Information about port license')
+        g = GaugeMetricFamily('ucs_port_license', 'Information about port license',
+                              labels=['server', 'port_name'])
```

Comment:
for more overview please explicitly specify labels here: https://github.com/sapcc/ucs-exporter/pull/2/files#diff-212dd652ba34a590784916f12488e3ddR30
`server= .., port_name=`